README.md

```diff
@@ -141,13 +141,15 @@ company, sub-projects within a large project such as Linux and Eclipse, or the o
 2. Second level: data sources and metadata
 3. Third level: data source URLs
 
-There are some filters, labels, and a special section:
+There are some filters, labels, and a special section and authentication:
```

Comment:
I feel this isn't the right place to provide this information. IIUC, this section basically explains the different options of the `projects.json` file. But, our motive is to explain about the a use case for git data source.

Instead, we can provide this information under the git subsection

```
#### git [&uarr;](#supported-data-sources-)
Commits from Git

**Note:** If you want to analyze private git repositories, make sure you pass the credentials directly in the URL

- projects.json
``'
{
    "Chaoss": {
        "git": [
            "https://github.com/chaoss/grimoirelab-perceval",
            "https://<username>:<api-token>@github.com/chaoss/grimoirelab-sirmordred"
        ]
    }
}
``'
- setup.cfg
```